modules/EvseManager/manifest.yaml

```diff
@@ -6,6 +6,9 @@ config:
   connector_id:
     description: Connector id of this evse manager
     type: integer
+  connector_type:
+    description: The connector type of this evse manager
```

Comment:
In this description you should point to the enum of available values for this parameter